docs/src/main/sphinx/sql/alter-materialized-view.rst

```diff
@@ -19,6 +20,22 @@ materialized view does not exist. The error is not suppressed if the
 materialized view does not exist, but a table or view with the given name
 exists.
 
+.. _alter-materialized-view-set-properties:
+
+SET PROPERTIES
+^^^^^^^^^^^^^^
+
+The ``ALTER MATERALIZED VIEW SET PROPERTIES``  statement followed by some number
+of ``property_name`` and ``expression`` pairs applies the specified properties
+and values to a materialized view. Excluding a property from this statement
```

Comment:
`Excluding a property` -> `Omitting an already set property of the materialized view`